sites/docs/docs/plugins/creating-a-plugin/components.md

```diff
@@ -0,0 +1,43 @@
+# Creating a component plugin 
+
+## Starting from a Template
+
+The easiest way to get started is from the example component library [**on GitHub**](https://github.com/evidence-dev/labs), with a live demo of the components [here](https://labs.evidence.dev).
```

Comment:
When we create the template component lib for product hunt we should link this instead